ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml

```diff
@@ -164,6 +164,10 @@ tests:
   secret:
     mount_path: /tmp/secret
     name: integreatly-operator-codecov-token
+- as: lint
+  commands: GOLANGCI_LINT_CACHE=/tmp/golangci-cache make test/lint
```

Comment:
Added the `GOLANGCI_LINT_CACHE` environment variable because the ci runner falls back to a cache directory which we cannot write to: [job failure](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/33258/rehearse-33258-pull-ci-integr8ly-integreatly-operator-master-lint/1587477888083431424)